frontend/src/api/MyUserApi.tsx

```diff
@@ -8,16 +9,19 @@ type CreateUserRequest = {
 };
 
 export const useCreateMyUser = () => {
+  const { getAccessTokenSilently } = useAuth0();
   const createMyUserRequest = async (user: CreateUserRequest) => {
+    const accessToken = await getAccessTokenSilently();
     const response = await fetch(`${API_BASE_URL}/api/my/user`, {
       method: 'POST',
       headers: {
+        Authorization: `Bearer ${accessToken} `,
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/16#issuecomment-2365073767>#8f1c23</a><span> - 09/21/2024, 09:15 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Extra space in the Authorization header when setting the access token
</div>
<div id="fix">
<b>Fix:</b> Remove the extra space after the access token in the Authorization header.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -18,1 +18,1 @@
 -        Authorization: 'Bearer ${accessToken} ',
 +        Authorization: 'Bearer ${accessToken}',
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged